low_coil_signal_threshold: 0.5
high_coil_signal_threshold: 0.95
derivative_sample_time: 0.1
number_of_derivatives: 6

std_radius: 0.5
